sdcm/utils.py

```diff
@@ -196,6 +212,45 @@ def upload_file(cls, file_path):
             logger.debug("Unable to upload to S3: %s" % e)
             return ""
 
+    @classmethod
+    def download_files_by_testid_from_bucket(cls, bucket_name, testid):
+        filenames = cls.find_filename_by_testid(bucket_name, testid)
+        for f in filenames:
+            cls.download_file(f)
+
+    @staticmethod
+    def download_file(link):
+        resp = requests.get(link)
+        try:
+            if resp.status_code == 200:
+                file_path = os.path.basename(os.path.dirname(link))
+                if not os.path.exists(file_path):
+                    os.mkdir(file_path)
+                with open(os.path.join(file_path, os.path.basename(link)), 'wb') as fh:
+                    fh.write(resp.content)
+        except Exception as details:
+            logger.warning("File {} is not downloaded by reason: {}".format(file_path), details)
+
+
+def list_logs_by_test_id(test_id):
+    bucket_name = 'cloudius-jenkins-test'
+    log_types = ['db-cluster', 'monitor-set', 'prometheus', 'grafana', 'job']
+    results = []
+
+    if not test_id:
+        return results
+
+    log_files = S3Storage.find_files_by_testid_in_bucket(bucket_name, test_id)
+    for log_file in log_files:
+        for log_type in log_types:
+            if log_type in log_file:
+                results.append({"file_path": log_file,
+                                "type": log_type,
+                                "link": "https://{}.s3.amazonaws.com/{}".format(bucket_name, log_file)})
```

Comment:
no need to pass the bucket name all over the methods

Reply:
Fixed. Idea was, to use with any bucket